clip_as_rnn/SAM/SAM.py

```diff
@@ -0,0 +1,193 @@
+import os
```

Comment:
There might be something wrong with your pull request. 81 files are being modified.